test/integrationtests/test_selene_api.py

```diff
@@ -14,15 +14,16 @@
 #
 import unittest
 from copy import copy
-import selene_api
+import ovos_backend_client
```

Comment:
Should this test the deprecated API module in `mycroft.deprecated.api` too? In general, should we test that deprecated code, maybe in a different test suite?

Reply:
the deprecated module is about compat with mycroft-core, not with ourselves, for that we have semver,

 this is a new lib that was in core for very little time so its ok to simply import from it, thats not ovos-core public api, the question is this test even makes sense here anymore?